src/AppEndToEnd.test.tsx

```diff
@@ -6,6 +6,9 @@ import electionSample from './data/electionSample.json'
 import App, { electionKey, mergeWithDefaults } from './App'
 import { CandidateContest, Election, YesNoContest } from './config/types'
 
+// turn off voterBallotTracker for these tests
+electionSample.voterBallotTracker = false
```

Comment:
Sorry I didn't catch this sooner, but it seems that `voterBallotTracker` is an option which can be configured on/off and should be placed into [electionDefaults.json](https://github.com/votingworks/bmd/blob/master/src/data/electionDefaults.json) with  default of `false`. Let's make this change.

Reply:
I'm not sure that's right, though tell me what you think: it's meant to be something that's found in `election.json`, not something that is later configurable. `election.json` is the final word. This test line is meant to do the end-to-end test without ballot tracking without also having to duplicate the sample file.